.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
```

Comment:
Hi, @odra is this env fine for this project? Please, could you check this PR?

Reply:
yeah it's fine

Reply:
👍